backends/concrete-cpu/implementation/src/c_api/secret_key.rs

```diff
@@ -211,6 +211,48 @@ pub unsafe extern "C" fn concrete_cpu_decompress_seeded_lwe_ciphertext_u64(
     });
 }
 
+#[no_mangle]
+pub unsafe extern "C" fn concrete_cpu_serialize_lwe_secret_key_u64(
+    lwe_sk: *const u64,
+    lwe_dimension: usize,
+    out_buffer: *mut u8,
+    out_buffer_len: usize,
+) -> usize {
+    let lwe_sk = LweSecretKey::from_container(slice::from_raw_parts(
```

Comment:
The idea was to make sure the serialized data doesn't go over the buffer by first checking its size. However, it might be enough to call `size_of_val` on the serialized value. I will check this, and if it's the case, then I will change the pattern of serializing data as it has also been used in serializing fheuint